build.gradle

```diff
@@ -399,8 +404,8 @@ startScripts {
   }
 
   doLast {
-    unixScript.text = unixScript.text.replace('ETHSIGNER_HOME', '\$APP_HOME')
-    windowsScript.text = windowsScript.text.replace('ETHSIGNER_HOME', '%~dp0..')
+    unixScript.text = unixScript.text.replace(repositoryName.toUpperCase() + "_HOME", '\$APP_HOME')
```

Comment:
may I suggest, right where you defined repositoryName, also define `HOME_ENV=repositoryName.toUpperCase() + "_HOME"` which would allow readability and easier maintenance.

Reply:
done - nicer.